python-sdk/nuscenes/eval/detection/data_classes.py

```diff
@@ -262,6 +263,25 @@ def __eq__(self, other):
             eq = eq and np.array_equal(getattr(self, key), getattr(other, key))
         return eq
 
+    @property
+    def max_recall_ind(self):
+        """ Returns index of max recall achieved. """
+
+        # Last instance of confidence > 0 is index of max achieved recall.
+        non_zero = np.nonzero(self.confidence)
```

Comment:
Maybe cleaner to do `non_zero, _ = np.nonzero(self.confidence)`? That way you can get rid of that pesky `[0]` in what follows.